test/spec/app_spec.rb

```diff
@@ -632,63 +619,58 @@
 
     it "should not add a duplicated tag from command line", bin3: true do
       media_pg = api_create_team_project_and_claim_and_redirect_to_media_page
-
       new_tag = Time.now.to_i.to_s
+      old = @driver.find_elements(:class,"annotations__list-item").length
 
       # Validate assumption that tag does not exist
       expect(media_pg.has_tag?(new_tag)).to be(false)
 
       # Try to add from command line
       media_pg.add_annotation("/tag #{new_tag}")
-      Selenium::WebDriver::Wait.new(timeout: 10).until { media_pg.has_tag?(new_tag) } # TODO: wait inside MediaPage
+      old = wait_for_size_change(old, "annotations__list-item", :class)
+      media_pg.has_tag?(new_tag)
       expect(media_pg.has_tag?(new_tag)).to be(true)
 
       # Try to add duplicate from command line
       media_pg.add_annotation("/tag #{new_tag}")
 
       # Verify that tag is not added and that error message is displayed
       expect(media_pg.tags.count(new_tag)).to be(1)
+      sleep 3
```

Comment:
Can't we wait for the `.message` element here instead of `sleep`ing?

Reply:
Good idea. Will try it.

Reply:
Using the analyzer I didn't find a name for the message code, like ".message". 
So I was not able to insert a code to wait for it, since the div/class has no name.